test/chi/js/chi-drawer.pug

```diff
@@ -0,0 +1,134 @@
+---
+title: Enterprise Chi-Drawer test case Pug
+---
+script(src='/js/chi.js')
+
+button#drawer-trigger-1.chi-button.-icon.chi-drawer__trigger(data-target='#drawer-1', aria-label='Toggle navigation')
+  .chi-button__content
+    i.chi-icon.-sm--2.icon-menu
+
+.chi-backdrop.-closed
+  .chi-backdrop__wrapper
+    #drawer-1.chi-drawer.-left
+      .chi-drawer__header
+        button.chi-button.-icon.-close(aria-label='Close')
+          .chi-button__content
+            i.chi-icon.icon-x
+      .chi-drawer__content
+        .-px--2.-text Drawer content here
+
+script.
```

Comment:
Let's move all the JS scripts to the bottom of the page inside one consolidated `script.`, please